examples/immersive_paris.html

```diff
@@ -0,0 +1,172 @@
+        <script src="js/GUI/dat.gui/dat.gui.min.js"></script>
+        <script src="js/GUI/GuiTools.js"></script>
+        <script src="js/proj4defs/2154.js"></script>
+        <script src="../dist/debug.js"></script>
+        <script type="text/javascript">
+            // Define initial camera position
+            var positionOnGlobe = { longitude: 2.33481381638492, latitude: 48.850602961052147, altitude: 50 };
+
+            // `viewerDiv` will contain iTowns' rendering area (`<canvas>`)
+            var viewerDiv = document.getElementById('viewerDiv');
+
+            var orientedImageLayer;
+
+            // Instanciate iTowns GlobeView
+            var view = new itowns.GlobeView(viewerDiv, positionOnGlobe, {
+                noControls: true,
+                handleCollision: false,
+                sseSubdivisionThreshold: 10,
+            });
+
+            // create Immersive control
+            view.controls = new itowns.ImmersiveControls(view, {
+                animationDuration: 50,
+            });
+
+            // limit camera far, to increase performance
+            view.camera.camera3D.far = 60;
+            view.camera.camera3D.near = 0.1;
+
+            // open camera fov
+            view.camera.camera3D.fov = 75;
+            
+            // disable atmosphere
+            view.atmosphere.visible = false;
+
+            // Add one imagery layer to the scene
+            // This layer is defined in a json file but it could be defined as a plain js
+            // object. See Layer* for more info.
+            itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(function _(config) {
+                config.source = new itowns.WMTSSource(config.source);
+                var layer = new itowns.ColorLayer('Ortho', config);
+                view.addLayer(layer).then(menuGlobe.addLayerGUI.bind(menuGlobe));
+            });
+
+            // Add two elevation layers.
+            // These will deform iTowns globe geometry to represent terrain elevation.
+            function addElevationLayerFromConfig(config) {
+                config.source = new itowns.WMTSSource(config.source);
+                var layer = new itowns.ElevationLayer(config.id, config);
+                view.addLayer(layer).then(menuGlobe.addLayerGUI.bind(menuGlobe));
+            }
+            itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addElevationLayerFromConfig);
+
+
+            function colorBuildings(properties) {
+                if (properties.id.indexOf('bati_remarquable') === 0) {
+                    return new itowns.THREE.Color(0x5555ff);
+                } else if (properties.id.indexOf('bati_industriel') === 0) {
+                    return new itowns.THREE.Color(0xff5555);
+                }
+                return new itowns.THREE.Color(0xeeeeee);
+            }
+
+            function altitudeBuildings(properties) {
+                return properties.z_min - properties.hauteur - 3;
```

Comment:
Why this value: 3?

Reply:
It's specific to this example. It add a delta of 3 meters, to be sure that the building start under the ground (define by the MNT).

Reply:
a comment will be welcome